guides/common/modules/proc_configuring-remote-execution-for-pull-client-on-project-server.adoc

```diff
@@ -1,38 +1,37 @@
 [id="configuring-remote-execution-for-pull-client-on-{project-context}-server_{context}"]
-= Configuring remote execution for pull client on {ProjectServer}
+= Configuring pull-based transport for remote execution
 
-By default, Remote Execution uses SSH as the transport mechanism for the Script provider.
-However, Remote Execution also offers pull-based transport, which you can use if your infrastructure prohibits outgoing connections from {Project} to hosts.
+By default, remote execution uses push-based SSH as the transport mechanism for the Script provider.
+If your infrastructure prohibits outgoing connections from {Project} to hosts, you can use remote execution with pull-based transport instead.
```

Comment:
Outgoing connections from smart proxies to hosts

Reply:
Are you sure? This is meant for Foreman server.

Reply:
A smart proxy (as a service) runs on Foreman server (as a machine)